pkg/services/mssql/all_in_one_arm_template.go

```diff
@@ -54,33 +30,35 @@ var allInOneARMTemplateBytes = []byte(`
 				{
 					"type": "firewallrules",
 					"name": "{{.Name}}",
-					"apiVersion": "[variables('SQLapiVersion')]",
+					"apiVersion": "2014-04-01-preview",
```

Comment:
I think you had mentioned this to me in passing-- just double-checking, however, that the `apiVersion` on the firewall rules _is_ supposed to be different from the `apiVersion` for the server (and that that `apiVersion` _is_ supposed to be different from the one for the database, for that matter).

Reply:
yep. I ran into issues using just the latest for everything and tried a few other combinations before I gave up and just fell back to what the portal produces if you use the `Automation Options` button